.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
So now it's completely empty don't you think you good to give have at least a hint on what we want the description to include.

```suggestion
<Describe what changes you made or in case of a bug fix what caused it how it was solved>

```